test/brick_seeker_web/features/searching_for_parts_test.exs

```diff
@@ -1,9 +1,5 @@
 defmodule BrickSeeker.SearchingForPartsTest do
-  use ExUnit.Case, async: false
-  use Wallaby.Feature
```

Comment:
Running mix test results in this locally.
> test/brick_seeker_web/features/autocomplete_test.exs:3: undefined function feature/3 (there is no such import)

More generally, using the Wallaby.Feature module allows for the use of the feature macro, which also manages taking screenshots automatically on failure. I've used a project-specific FeatureCase in the past, but it makes managing that part of feature tests a little bit harder. I think I could be convinced to use FeatureCase if the wallaby screenshot_on_failure tag could still be used automatically.

Reply:
To expand on this a bit, here's where the macro handles the screenshot on failure https://github.com/elixir-wallaby/wallaby/blob/main/lib/wallaby/feature.ex#L113. The feature macro also does the same sandbox checkout as the docs guide us to do manually if we don't want to use the macro (https://github.com/elixir-wallaby/wallaby/blob/main/lib/wallaby/feature.ex#L160-L179). In my opinion, there's not a lot of benefit to the FeatureCase other than limiting duplication of some boilerplate code (imports, alias, etc), and it comes at the expense of the screenshotting on failure behavior. I've thought about this one a lot, and I still don't like either tradeoff 😆 

Hoping more people weigh in on this discussion

Reply:
I had forgotten to call the feature case in that file so it wasn't compiling. I have fixed it below and added screenshots on failure to the config. there would be a little bit more to do to get it working in CI

Reply:
Ah, I see now how the feature module works. The import lets us still get the benefit of the screenshot_on_failure without using the macro. I am still a skeptic of the value of FeatureCase in general, but it does bring consistency alongside the other "Case" modules and it has the added benefit of things like setting the initial browser size, so I'm good to roll with it. The CI part of adding the screenshots to the build artifacts should already be there. I hadn't activated it yet, so thank you for doing that in a followup commit.